arlas-server-xpack/src/main/java/io/arlas/server/app/ArlasServerXpack.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
We may need to add the Elasticsearch license to the the header
@sylvaingaudan @mbarbet

Reply:
No longer needed.